Platform/Shared/VMContextMenuModifier.swift

```diff
@@ -56,7 +57,13 @@ struct VMContextMenuModifier: ViewModifier {
                 } label: {
                     Label("Run", systemImage: "play.fill")
                 }
+                Button {
+                    data.run(vm: vm, runAsSnapshot: true)
+                } label: {
+                    Label("Run as Snapshot", systemImage: "play")
```

Comment:
Let's distinguish the snapshot button by using the `play.rectangle` symbol instead.